src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_doing_flr_with_dtc_on.cs

```diff
@@ -50,27 +54,13 @@ public class RetryEndpoint : EndpointConfigurationBuilder
         {
             public RetryEndpoint()
             {
-                EndpointSetup<DefaultServer>(b => b
-                    .EnableFeature<FirstLevelRetries>())
-                    .WithConfig<TransportConfig>(c => c.MaxRetries = maxretries);
-            }
-
-            class ErrorNotificationSpy : IWantToRunWhenBusStartsAndStops
-            {
-                public Context Context { get; set; }
-
-                public Notifications Notifications { get; set; }
-
-                public Task Start(IMessageSession session)
-                {
-                    Notifications.Errors.MessageSentToErrorQueue += (sender, message) => Context.GaveUpOnRetries = true;
-                    return Task.FromResult(0);
-                }
-
-                public Task Stop(IMessageSession session)
+                EndpointSetup<DefaultServer>((b, context) =>
                 {
-                    return Task.FromResult(0);
-                }
+                    var scenarioContext = (Context) context.ScenarioContext;
+                    b.EnableFeature<FirstLevelRetries>();
+                    b.GetSettings().Get<Notifications>().Errors.MessageSentToErrorQueue += (sender, message) => scenarioContext.GaveUpOnRetries = true;
```

Comment:
I see no value in doing this right now, yes it is hidden, not perfect. But
we can improve this situation as an additive in 6.x